### PR TITLE
Add colab demo

### PR DESCRIPTION
**Pull request by AllentDan**
Failed to put another demo of deepseek-vl 1.3b. It was fixed in #1427

### CODE REVIEW

**Comment by lvhan028** [comment]
So, we should add deepseek-vl 1.3b colab demo after lmdeploy released, shouldn't we?

**Comment by AllentDan** [comment]
> So, we should add deepseek-vl 1.3b colab demo after lmdeploy released, shouldn't we?

Yes, this is the better way. Or, we can hack the codes of v0.3.0.

File: README.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ LMDeploy has developed two inference engines - [TurboMind](./docs/en/inference/t
 
 They differ in the types of supported models and the inference data type. Please refer to [this table](./docs/en/supported_models/supported_models.md) for each engine's capability and choose the proper one that best fits your actual needs.
 
-# Quick Start
+# Quick Start[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1Dh-YlSwg78ZO3AlleO441NF_QP2shs95#scrollTo=YALmXnwCG1pQ)
```
Comment (lvhan028) [inline README.md:118]: [!Open In Colab] 和 Quick Start 之间，留个空格吧

**Comment by lvhan028** [comment]
```
!pip install nest_asyncio
```
是不是等我们发新版之后，这个安装就不需要了呢？

**Comment by lvhan028** [comment]
> > So, we should add deepseek-vl 1.3b colab demo after lmdeploy released, shouldn't we?
> 
> Yes, this is the better way. Or, we can hack the codes of v0.3.0.

Let's choose the better way

**Comment by lvhan028** [comment]
这个 PR 延迟到 v0.4.0 发版之后。
等新版本发布后，验证 "pip install nest_asyncio" 是不是必须

**Comment by lvhan028** [comment]
Can we make offline jupter notebook files?

**Comment by AllentDan** [comment]
> Can we make offline jupter notebook files?

I thought the pipy package could be modified in colab but I did not test it yet.

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**